src/certificate.js

```diff
@@ -3,30 +3,30 @@ import parseJSON from './parser';
 import Verifier from './verifier';
 
 export default class Certificate {
-  constructor (certificateJson) {
-    if (typeof certificateJson !== 'object') {
+  constructor (certificateContent) {
```

Comment:
could call it `certificateDefinition`? That's mainly what I am doing in the BUV.

Reply:
Sure thing